api-service/webapp-demo/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "http://localhost:8080",
+    "target": "http://localhost:8083",
```

Comment:
I don't think you meant to commit this. I'll add an issue to fix the problem with ports for local development.

Reply:
https://github.com/conversationai/conversationai-crowdsource/issues/11

Reply:
Right, thanks for catching this! I'll revert it.